dev/release/verify-release-candidate.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
I've always seen the former approach of `#!/bin/bash`. For my own edification, why this change?

Reply:
My mental model for `env` is basically "search $PATH for this executable and use it".

The pattern is more commonly used for things like python, i.e. `#!/usr/bin/env python` because then the script will automatically use whatever virtual environment you have activated.
 
It's less commonly used for `bash`, but I use a pretty niche operating system (NixOS) which doesn't use `/bin/bash`.

```console
❯ which bash
/nix/store/lmv9b9jbhv68kp5pg467rlgg71lf0zyl-bash-interactive-5.2p26/bin/bash

❯ ls -l /usr/bin
total 4
lrwxrwxrwx 1 root root 65 Oct  3 06:58 env -> /nix/store/0kg70swgpg45ipcz3pr2siidq9fn6d77-coreutils-9.5/bin/env

❯ ls -l /bin
total 4
lrwxrwxrwx 1 root root 74 Oct  3 06:58 sh -> /nix/store/qqz0gj9iaidabp7g34r2fb9mds6ahk8i-bash-interactive-5.2p32/bin/sh
```